Client/Frontend/ContentBlocker/ContentBlockerSettingViewController.swift

```diff
@@ -70,6 +71,8 @@ class ContentBlockerSettingViewController: ContentBlockerSettingsTableView {
                 ContentBlockerHelper.prefsChanged()
 
                 LeanplumIntegration.sharedInstance.track(eventName: .trackingProtectionSettings, withParameters: ["Enabled option": option.rawValue as AnyObject])
+                Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.appSettingChange, object: TelemetryEventObject.trackingProtectionEnabled, value: option.rawValue)
```

Comment:
Can probably shorthand these enum values (i.e. `category: .action, method: .appSettingChange`, etc.)

Reply:
Oh wait, they're not enums, are they?

Reply:
Nevermind... back to bed 🤒